engine/src/main/java/org/terasology/world/chunks/internal/ChunkImpl.java

```diff
@@ -64,6 +65,9 @@
     private static final DecimalFormat PERCENT_FORMAT = new DecimalFormat("0.##");
     private static final DecimalFormat SIZE_FORMAT = new DecimalFormat("#,###");
 
+    private final int verticalChunkMeshSegments = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
+    private final int chunkHalfHeight = ChunkConstants.SIZE_Y / verticalChunkMeshSegments / 2;
```

Comment:
You mean *every* visible face of a single block is *always* 2 triangles and 4 vertices? And the VERTICAL SEGMENTS value of 2 would split that face vertically in two half, resulting in 4 triangles and 6-8 vertices?

Anyway, I understand the VERTICAL_SEGMENTS value needs to go. I can take care of this in another PR.